src/ida_ipython.cpp

```diff
@@ -3,6 +3,8 @@
 #include "pro.h"
 #include "ida.hpp"
 
+#include "persist.c"
```

Comment:
I think this should be including the header file, not the c file. We could either just write persist in C++ (just change the file name to .cpp) and include the header file, or wrap the include in an extern "C"